### ISSUE TITLE
Test for the resume

### ISSUE THREAD

**Issue by icaiyu** [embulk/embulk#722]
Hello, I am a new developer of the Embulk plugin, Is there any way to test the Resume support of the plugin? By the way, to support the resume of the data loading, all my plugins should support the resume? Including the input plugin, the parser, and the output plugin? Waiting for your answer. Thank you.

**Comment by dmikurube** [comment]
Thanks for your interests on Embulk!

For testing, do you have any ideas? @muga 

It'd depend on the plugins, the data source, the destination and the situation, but I guess resuming support just in input and sometimes in output would work in many cases.

Not needed in parser, filter or other plugins since they're not data source/destination and they're stateless. Resuming in output may not be required if no failures are expected for output.

**Comment by muga** [comment]
@icaiyu @dmikurube Good point. We don't have anything to check if a plugin supports resume functionality or not so much.

For now, almost all of plugins (plugin developers) may describe that plugins support resume in README.md. But, some plugins doesn't show such descriptions. We don't have a guideline about it. And even though some plugins describe that they have resume functionality in README files, we don't know that their resume correctly work or not.

**Comment by icaiyu** [comment]
Thank you for your reply. Could you give some tips so that I can test the resume support functionality. For example, I want to the test the resume support of the embulk standard local file input plugin. I try to stop the embulk by pressing ctl+c but there are nothing in the diff.yml.
